scripts/glutton/update/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "glutton-update-bin"
+version = "1.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "Script to update Glutton parachains storage values"
+
+
+[[bin]]
+name = "glutton-update"
+path = "src/main.rs"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.4.0", default-features = false }
+subxt = { version = "0.29.0", features = ["unstable-metadata"] }
```

Comment:
The alternative I initially used was to query the `nonce` from `frame_system` storage, but with the `["unstable-metadata"]` feature I could directly use the runtime API to query it. All this is coming from the fact that the Shell client does not implement the `system_accountNextIndex` method.

In case it breaks in the future I can always bring back the former approach. I'll keep it in mind, thanks for the heads up.